src/transport/Session.h

```diff
@@ -254,7 +254,7 @@ class Session
     // GetAckTimeout is the estimate for how long it could take for the other
     // side to receive our message (accounting for our MRP retransmits if it
     // gets lost) and send a response.
-    virtual System::Clock::Milliseconds32 GetAckTimeout() const = 0;
+    virtual System::Clock::Milliseconds32 GetAckTimeout(bool isPeerActive) const = 0;
```

Comment:
What does this boolean really mean?  Is it a "we know the peer is active via some information not available to the session"?  Or is it a "the MRP machinery on our side will assume the peer is active regardless of what the session state is"?

I suspect it's the latter of those two, but please double-check.  The naming, documentation (which needs to exist) and actual setting of this parameter need to match the intended meaning.

Reply:
yes, the latter should be right after checking around the code. updated. thanks